tests/jvm/src/test/scala/scala/meta/tests/metacp/ClasspathLookupSuite.scala

```diff
@@ -0,0 +1,61 @@
+package scala.meta.tests.metacp
+
+import org.scalatest.FunSuite
+import scala.meta.internal.io.PathIO
+import scala.meta.internal.metacp.ClasspathLookup
+import scala.meta.io.AbsolutePath
+import scala.meta.io.Classpath
+import scala.meta.tests.BuildInfo
+import scala.meta.tests.semanticdb.ManifestMetacp
+
+class ClasspathLookupSuite extends FunSuite {
```

Comment:
Very thorough tests! 👍

Reply:
Was gratifying to see it catch a windows bug!